libs/books/data-access/src/lib/+state/reading-list.effects.spec.ts

```diff
@@ -8,32 +8,34 @@ import { SharedTestingModule } from '@tmo/shared/testing';
 import { ReadingListEffects } from './reading-list.effects';
 import * as ReadingListActions from './reading-list.actions';
 import { API_PATH } from '../constants';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
```

Comment:
Do you really need to change this file?

As I see there is no test case updated. If not required please avoid such kind of changes.